Update customize-ui-with-html.md (#110006)
Azure/azure-content: articles/active-directory-b2c/customize-ui-with-html.md

1. Under **Data storage** in the left-hand menu, select **Containers**.
1. Select **+ Container**.
1. For **Name**, enter *root*. The name can be a name of your choosing, for example *contoso*, but we use *root* in this example for simplicity.

> 1. For **Public access level**, select **Blob**. (**Please note that selecting Blob will permit anonymous public read-only access for this container.**)

Reviewer: ```suggestion
1. For **Public access level**, select **Blob**. By selecting the **Blob** option, you allow an anonymous public read-only access for this container.
```

Reviewer: @isamsaroia Thanks for your contribution!

Please see our suggestion.
